### ISSUE TITLE
passwordless ssh not working

### ISSUE THREAD

**Issue by DavidBiesack** [PowerShell/Win32-OpenSSH#409]
I installed the [3.0.0 release](https://github.com/PowerShell/Win32-OpenSSH/releases/tag/v0.0.3.0) on Win 7 Pro, followed the [install instructions](https://github.com/PowerShell/Win32-OpenSSH/wiki/Install-Win32-OpenSSH) and ran the `reg` command [here](https://github.com/PowerShell/Win32-OpenSSH/issues/189) and verified my file permissions (using Cygwin ls -l) : 

```
drwx------+ 1 myuserid Domain Users   0 Nov 15 10:15 /cygdrive/c/users/myuserid/.ssh
-rw-------+ 1 myuserid Domain Users 395 Nov 15 10:11 /cygdrive/c/users/myuserid/.ssh/authorized_keys

```
I've copied my `id_rsa.pub` public key (which works on other systems) into `c:/users/myuserid/.ssh/authorized_keys`

My `c:\openssh\sshd_config` contains 

```
RSAAuthentication yes
PubkeyAuthentication yes
```

I rebooted when instructed and also tried restarting ssh-agent and sshd etc.

Yet when I ssh from a Linux workstation, I'm still prompted for a password each time. Log below.

Can someone spot what's wrong (or are the install instructions omitting something? 

Correct PowerShell commands to set the permissions on .ssh and .ssh/authorized_keys would be useful in the install/setup instructions, as I'm using Cygwin but can't verify if the permissions are truly correct.

```
$ ssh -v aaaaaaaaa.bbbbbbb.sas.com xxxxxxxxxxxxxxxxxx
debug1: Reading configuration data /u/myuserid/.ssh/config
debug1: /u/myuserid/.ssh/config line 9: Applying options for *
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: /etc/ssh/ssh_config line 19: Applying options for *
debug1: Connecting to aaaaaaaaa.bbbbbbb.sas.com [10.23.11.92] port 22.
debug1: Connection established.
debug1: identity file /u/myuserid/.ssh/id_rsa type 1
debug1: key_load_public: No such file or directory
debug1: identity file /u/myuserid/.ssh/id_rsa-cert type -1
debug1: Enabling compatibility mode for protocol 2.0
debug1: Local version string SSH-2.0-OpenSSH_7.2p2 Ubuntu-4ubuntu2.1
debug1: Remote protocol version 2.0, remote software version OpenSSH_7.1p1 Microsoft_Win32_port_with_VS
debug1: match: OpenSSH_7.1p1 Microsoft_Win32_port_with_VS pat OpenSSH* compat 0x04000000
debug1: Authenticating to aaaaaaaaa.bbbbbbb.sas.com:22 as 'myuserid'
debug1: SSH2_MSG_KEXINIT sent
debug1: SSH2_MSG_KEXINIT received
debug1: kex: algorithm: curve25519-sha256@libssh.org
debug1: kex: host key algorithm: ssh-rsa
debug1: kex: server->client cipher: chacha20-poly1305@openssh.com MAC: <implicit> compression: none
debug1: kex: client->server cipher: chacha20-poly1305@openssh.com MAC: <implicit> compression: none
debug1: expecting SSH2_MSG_KEX_ECDH_REPLY
debug1: Server host key: ssh-rsa SHA256:K34t+i98JUxmuDx/oyCx71cxr+W5FHNm6O2qbbQHhMU
debug1: Host 'aaaaaaaaa.bbbbbbb.sas.com' is known and matches the RSA host key.
debug1: Found key in /u/myuserid/.ssh/known_hosts:422
debug1: rekey after 134217728 blocks
debug1: SSH2_MSG_NEWKEYS sent
debug1: expecting SSH2_MSG_NEWKEYS
debug1: rekey after 134217728 blocks
debug1: SSH2_MSG_NEWKEYS received
debug1: SSH2_MSG_SERVICE_ACCEPT received
debug1: Authentications that can continue: publickey,password,keyboard-interactive
debug1: Next authentication method: publickey
debug1: Offering RSA public key: /u/myuserid/.ssh/id_rsa
debug1: Server accepts key: pkalg ssh-rsa blen 279
debug1: Authentications that can continue: publickey,password,keyboard-interactive
debug1: Next authentication method: keyboard-interactive
debug1: Authentications that can continue: publickey,password,keyboard-interactive
debug1: Next authentication method: password
myuserid@aaaaaaaaa.bbbbbbb.sas.com's password: xxxxxxxxxxx
debug1: Authentication succeeded (password).
Authenticated to aaaaaaaaa.bbbbbbb.sas.com ([xx.xx.xx.xx]:22).
debug1: channel 0: new [client-session]
debug1: Requesting no-more-sessions@openssh.com
debug1: Entering interactive session.
debug1: pledge: network
debug1: client_input_global_request: rtype hostkeys-00@openssh.com want_reply 0
...
debug1: Sending command: xxxxxxxxxxxxxxxxxx
```

`sshd -d` debug log shows:

```
debug1: Client protocol version 2.0; client software version OpenSSH_7.2p2 Ubuntu-4ubuntu2.1
debug1: match: OpenSSH_7.2p2 Ubuntu-4ubuntu2.1 pat OpenSSH* compat 0x04000000
debug1: Enabling compatibility mode for protocol 2.0
debug1: Local version string SSH-2.0-OpenSSH_7.1p1 Microsoft_Win32_port_with_VS
debug1: list_hostkey_types: ssh-rsa,ecdsa-sha2-nistp256,ssh-ed25519
debug1: SSH2_MSG_KEXINIT sent
debug1: SSH2_MSG_KEXINIT received
debug1: kex: algorithm: curve25519-sha256@libssh.org
debug1: kex: host key algorithm: ssh-rsa
debug1: kex: client->server cipher: chacha20-poly1305@openssh.com MAC: <implicit> compression: none
debug1: kex: server->client cipher: chacha20-poly1305@openssh.com MAC: <implicit> compression: none
debug1: expecting SSH2_MSG_KEX_ECDH_INIT
debug1: SSH2_MSG_NEWKEYS sent
debug1: expecting SSH2_MSG_NEWKEYS
debug1: SSH2_MSG_NEWKEYS received
debug1: KEX done
debug1: userauth-request for user myuserid service ssh-connection method none
debug1: attempt 0 failures 0
Failed none for myuserid from aa.bb.cc.dd port 42146 ssh2
debug1: userauth-request for user myuserid service ssh-connection method publickey
debug1: attempt 1 failures 0
debug1: test whether pkalg/pkblob are acceptable
Postponed publickey for myuserid from aa.bb.cc.dd port 42146 ssh2
debug1: userauth-request for user myuserid service ssh-connection method publickey
debug1: attempt 2 failures 0
debug1: auth agent did not authorize client myuserid
Failed publickey for myuserid from aa.bb.cc.dd port 42146 ssh2: RSA SHA256:g/NZaQYcc3Ue5hf+WbTjaQ+g+2HCYFYNWSTPn6qd4QI
debug1: userauth-request for user myuserid service ssh-connection method keyboard-interactive
debug1: attempt 3 failures 1
debug1: keyboard-interactive devs
debug1: auth2_challenge: user=myuserid devs=
debug1: kbdint_alloc: devices ''
Failed keyboard-interactive for myuserid from aa.bb.cc.dd port 42146 ssh2
```

**Comment by manojampalam** [comment]
key auth is done in ssh-agent. See wiki for trouble shooting instructions and how to get ssh-agent traces

**Comment by DavidBiesack** [comment]
I see in ssh-agent.log

```
5924 09:56:53 947 debug1: unable to generate token for user myuserid
```
and in sshd.log
```
4416 09:56:53 947 debug1: auth agent did not authorize client userid
4416 09:56:53 947 debug2: userauth_pubkey: authenticated 0 pkalg ssh-rsa
4416 09:56:53 947 Failed publickey for userid from 10.23.16.105 port 54396 ssh2: RSA SHA256:g/NZaQYcc3Ue5hf+WbTjaQ+g+2HCYFYNWSTPn6qd4QI
4416 09:56:53 947 debug3: userauth_finish: failure partial=0 next methods="publickey,password,keyboard-interactive"
```

I'm not sure what this means or how to troubleshoot this (I looked at the wiki page to set DEBUG3 level) 
Here is the full `ssh-agent.log` for the client connection attempt:
```
8036 09:56:30 962 agent_start pid:8036, dbg:0, child:0, pipe:0
8036 09:56:51 853 client pid 4416 connected
8036 09:56:51 884 debug1: spawned worker 5924 for agent client pid 4416 
5924 09:56:53 947 agent_start pid:5924, dbg:0, child:1, pipe:348
5924 09:56:53 947 debug3: connection io 000000000031EA50 #bytes:0 state:0
5924 09:56:53 947 debug3: connection io 000000000031EA50 #bytes:4 state:1
5924 09:56:53 947 debug3: connection io 000000000031EA50 #bytes:956 state:2
5924 09:56:53 947 debug2: client type: 3
5924 09:56:53 947 debug1: process agent request type 100
5924 09:56:53 947 debug1: LsaLogonUser failed 1
5924 09:56:53 947 debug1: unable to generate token for user myuserid
5924 09:56:53 947 debug3: connection io 000000000031EA50 #bytes:5 state:3
```

Here is the full sshd.log for the same:

```
6308 09:56:37 837 debug2: signal() sig:2, handler:0000000000000001
6308 09:56:37 837 debug1: socket:220, io:000000000050FB30, fd:3 
6308 09:56:37 837 debug3: w32_fcntl fd:3
6308 09:56:37 837 debug2: fd 3 setting O_NONBLOCK
6308 09:56:37 837 debug3: w32_fcntl fd:3
6308 09:56:37 837 debug3: w32_setsockopt fd:3
6308 09:56:37 837 debug3: sock_set_v6only: set socket 3 IPV6_V6ONLY
6308 09:56:37 837 debug3: w32_setsockopt fd:3
6308 09:56:37 837 debug3: w32_fcntl fd:3
6308 09:56:37 837 debug1: Bind to port 22 on ::.
6308 09:56:37 837 debug3: w32_bind fd:3
6308 09:56:37 837 debug3: w32_listen fd:3
6308 09:56:37 837 Server listening on :: port 22.
6308 09:56:37 837 debug1: socket:364, io:000000000050E870, fd:4 
6308 09:56:37 837 debug3: w32_fcntl fd:4
6308 09:56:37 837 debug2: fd 4 setting O_NONBLOCK
6308 09:56:37 837 debug3: w32_fcntl fd:4
6308 09:56:37 837 debug3: w32_setsockopt fd:4
6308 09:56:37 837 debug3: w32_fcntl fd:4
6308 09:56:37 837 debug1: Bind to port 22 on 0.0.0.0.
6308 09:56:37 837 debug3: w32_bind fd:4
6308 09:56:37 837 debug3: w32_listen fd:4
6308 09:56:37 837 Server listening on 0.0.0.0 port 22.
6308 09:56:37 837 debug2: signal() sig:6, handler:000000013F9CE540
6308 09:56:37 837 debug2: signal() sig:3, handler:000000013F9CD100
6308 09:56:37 837 debug2: signal() sig:8, handler:000000013F9CE620
6308 09:56:37 837 debug2: signal() sig:7, handler:000000013F9CE620
6308 09:56:37 837 debug3: w32_select fd:3
6308 09:56:37 837 debug3: w32_select fd:4
6308 09:56:37 837 debug3: Total in fds:2
6308 09:56:37 837 debug2: on_select - io:000000000050FB30 type:1 rd:1
6308 09:56:37 837 debug3: acceptEx - io:000000000050FB30
6308 09:56:37 837 debug2: on_select - io:000000000050E870 type:1 rd:1
6308 09:56:37 853 debug3: acceptEx - io:000000000050E870
6308 09:56:37 853 debug3: wait() on 0 events and 0 children
6308 09:56:37 853 debug3: wait() on 2 events and 0 children
6308 09:56:49 665 debug3: select - returning 1
6308 09:56:49 665 debug3: w32_accept fd:4
6308 09:56:49 665 debug3: accept - io:000000000050E870
6308 09:56:49 665 debug2: accept io:000000000050E920
6308 09:56:49 665 debug1: socket:376, io:000000000050E920, fd:5 
6308 09:56:49 665 debug3: w32_fcntl fd:5
6308 09:56:49 665 debug3: fd 5 is not O_NONBLOCK
6308 09:56:49 665 debug1: pipe - read end: handle:000000000000017C, io:00000000004FC860, fd:6
6308 09:56:49 665 debug1: pipe - write end: handle:0000000000000180, io:00000000004FC910, fd:7
6308 09:56:49 665 debug1: Remote Handle 0000000000000178
6308 09:56:49 681 debug3: w32_close fd:5
6308 09:56:49 681 debug1: close - io:000000000050E920, type:1, fd:5, table_index:5
6308 09:56:49 681 debug2: close - io:000000000050E920
6308 09:56:49 697 debug3: w32_close fd:-1
6308 09:56:49 697 debug1: w32_close ERROR: bad fd: -1
6308 09:56:49 697 debug1: Register child 0000000000000188 pid 4416, 0 zombies of 0
6308 09:56:49 697 debug1: Forked child 4416.
6308 09:56:49 697 debug3: w32_close fd:7
6308 09:56:49 697 debug1: close - io:00000000004FC910, type:2, fd:7, table_index:7
6308 09:56:49 697 debug2: fileclose - pio:00000000004FC910
6308 09:56:49 697 debug3: w32_close fd:5
6308 09:56:49 697 debug1: w32_close ERROR: bad fd: 5
6308 09:56:49 697 debug3: w32_select fd:3
6308 09:56:49 697 debug3: w32_select fd:4
6308 09:56:49 697 debug3: w32_select fd:6
6308 09:56:49 697 debug3: Total in fds:3
6308 09:56:49 697 debug2: on_select - io:000000000050FB30 type:1 rd:1
6308 09:56:49 697 debug2: on_select - io:000000000050E870 type:1 rd:1
6308 09:56:49 697 debug3: acceptEx - io:000000000050E870
6308 09:56:49 697 debug2: ReadFileEx io:00000000004FC860
6308 09:56:49 697 debug3: wait() on 0 events and 1 children
6308 09:56:49 697 debug3: wait() on 2 events and 1 children
4416 09:56:51 853 debug2: signal() sig:2, handler:0000000000000001
4416 09:56:51 853 debug2: signal() sig:6, handler:000000013F9CE540
4416 09:56:51 853 debug2: signal() sig:3, handler:000000013F9CD100
4416 09:56:51 853 debug2: signal() sig:8, handler:000000013F9CE620
4416 09:56:51 853 debug2: signal() sig:7, handler:000000013F9CE620
4416 09:56:51 853 debug1: remote channel 376
4416 09:56:51 853 debug3: w32_fcntl fd:3
4416 09:56:51 853 debug3: w32_fcntl fd:3
4416 09:56:51 853 debug3: alarm() 0 secs
4416 09:56:51 853 debug2: signal() sig:4, handler:0000000000000000
4416 09:56:51 853 debug2: signal() sig:6, handler:0000000000000000
4416 09:56:51 853 debug2: signal() sig:8, handler:0000000000000000
4416 09:56:51 853 debug2: signal() sig:7, handler:0000000000000000
4416 09:56:51 853 debug2: signal() sig:3, handler:0000000000000000
4416 09:56:51 853 debug2: signal() sig:0, handler:0000000000000000
4416 09:56:51 853 debug3: w32_getpeername fd:3
4416 09:56:51 853 debug3: w32_setsockopt fd:3
4416 09:56:51 853 debug3: w32_getpeername fd:3
4416 09:56:51 853 debug3: w32_getpeername fd:3
4416 09:56:51 853 debug3: w32_getsockopt fd:3
4416 09:56:51 853 debug3: w32_getpeername fd:3
4416 09:56:51 853 debug3: w32_getsockname fd:3
4416 09:56:51 853 debug3: w32_getsockname fd:3
4416 09:56:51 853 Connection from 10.23.16.105 port 54396 on 10.23.11.92 port 22
4416 09:56:51 853 debug2: signal() sig:4, handler:000000013F9CB3A0
4416 09:56:51 853 debug3: alarm() 120 secs
4416 09:56:51 853 debug3: w32_write fd:3
4416 09:56:51 853 debug2: send - io:000000000025AC40
4416 09:56:51 853 debug2: send - WSASend() returned 0, APC scheduled io:000000000025AC40
4416 09:56:51 853 debug2: WSASendCB - io:000000000025AC40, pending_state:1, error:0, sent:52 of remaining:52
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug3: WSARecv - pio: 000000000025AC40
4416 09:56:51 853 debug2: WSARecv - WSARecv() returned 0, io:000000000025AC40
4416 09:56:51 853 debug2: recv - Letting APC to execute, io:000000000025AC40
4416 09:56:51 853 debug2: WSARecvCompletionCB - io:000000000025AC40, pending_state:1, flags:0, error:0, received:41
4416 09:56:51 853 debug3: recv - socket in blocking mode, io:000000000025AC40
4416 09:56:51 853 debug2: recv - (2) returning 1 bytes from completed IO, remaining:40, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:39, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:38, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:37, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:36, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:35, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:34, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:33, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:32, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:31, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:30, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:29, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:28, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:27, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:26, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:25, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:24, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:23, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:22, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:21, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:20, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:19, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:18, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:17, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:16, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:15, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:14, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:13, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:12, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:11, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:10, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:9, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:8, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:7, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:6, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:5, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:4, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:3, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:2, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:1, io:000000000025AC40
4416 09:56:51 853 debug3: w32_read fd:3
4416 09:56:51 853 debug3: recv - io:000000000025AC40
4416 09:56:51 853 debug2: recv - returning 1 bytes from prior completed IO, remaining:0, io:000000000025AC40
4416 09:56:51 853 debug1: Client protocol version 2.0; client software version OpenSSH_7.2p2 Ubuntu-4ubuntu2.1
4416 09:56:51 853 debug1: match: OpenSSH_7.2p2 Ubuntu-4ubuntu2.1 pat OpenSSH* compat 0x04000000
4416 09:56:51 853 debug1: Enabling compatibility mode for protocol 2.0
4416 09:56:51 853 debug1: Local version string SSH-2.0-OpenSSH_7.1p1 Microsoft_Win32_port_with_VS
4416 09:56:51 853 debug3: w32_fcntl fd:3
4416 09:56:51 853 debug2: fd 3 setting O_NONBLOCK
4416 09:56:51 853 debug3: w32_fcntl fd:3
4416 09:56:51 884 debug3: list_hostkey_types: ssh-dss key not permitted by HostkeyAlgorithms
4416 09:56:51 884 debug1: list_hostkey_types: ssh-rsa,ecdsa-sha2-nistp256,ssh-ed25519
4416 09:56:51 884 debug1: SSH2_MSG_KEXINIT sent
4416 09:56:51 884 debug3: w32_write fd:3
4416 09:56:51 884 debug2: send - io:000000000025AC40
4416 09:56:51 884 debug2: send - WSASend() returned 0, APC scheduled io:000000000025AC40
4416 09:56:51 884 debug2: WSASendCB - io:000000000025AC40, pending_state:1, error:0, sent:840 of remaining:840
4416 09:56:51 884 debug3: w32_select fd:3
4416 09:56:51 884 debug3: Total in fds:1
4416 09:56:51 884 debug2: on_select - io:000000000025AC40 type:1 rd:1
4416 09:56:51 884 debug3: WSARecv - pio: 000000000025AC40
4416 09:56:51 884 debug2: WSARecv - WSARecv() returned 0, io:000000000025AC40
4416 09:56:51 884 debug3: wait() on 0 events and 0 children
4416 09:56:51 884 debug2: WSARecvCompletionCB - io:000000000025AC40, pending_state:1, flags:0, error:0, received:1336
4416 09:56:51 884 debug3: select - returning 1
4416 09:56:51 884 debug3: w32_read fd:3
4416 09:56:51 884 debug3: recv - io:000000000025AC40
4416 09:56:51 884 debug2: recv - returning 1336 bytes from prior completed IO, remaining:0, io:000000000025AC40
4416 09:56:51 884 debug1: SSH2_MSG_KEXINIT received
4416 09:56:51 884 debug2: local server KEXINIT proposal
4416 09:56:51 884 debug2: KEX algorithms: curve25519-sha256@libssh.org,ecdh-sha2-nistp256,ecdh-sha2-nistp384,diffie-hellman-group-exchange-sha256,diffie-hellman-group14-sha1
4416 09:56:51 884 debug2: host key algorithms: ssh-rsa,ecdsa-sha2-nistp256,ssh-ed25519
4416 09:56:51 884 debug2: ciphers ctos: chacha20-poly1305@openssh.com,aes128-ctr,aes192-ctr,aes256-ctr
4416 09:56:51 884 debug2: ciphers stoc: chacha20-poly1305@openssh.com,aes128-ctr,aes192-ctr,aes256-ctr
4416 09:56:51 884 debug2: MACs ctos: umac-64-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512-etm@openssh.com,hmac-sha1-etm@openssh.com,umac-64@openssh.com,umac-128@openssh.com,hmac-sha2-256,hmac-sha2-512,hmac-sha1
4416 09:56:51 884 debug2: MACs stoc: umac-64-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512-etm@openssh.com,hmac-sha1-etm@openssh.com,umac-64@openssh.com,umac-128@openssh.com,hmac-sha2-256,hmac-sha2-512,hmac-sha1
4416 09:56:51 884 debug2: compression ctos: none,zlib@openssh.com
4416 09:56:51 884 debug2: compression stoc: none,zlib@openssh.com
4416 09:56:51 884 debug2: languages ctos: 
4416 09:56:51 884 debug2: languages stoc: 
4416 09:56:51 884 debug2: first_kex_follows 0 
4416 09:56:51 884 debug2: reserved 0 
4416 09:56:51 884 debug2: peer client KEXINIT proposal
4416 09:56:51 884 debug2: KEX algorithms: curve25519-sha256@libssh.org,ecdh-sha2-nistp256,ecdh-sha2-nistp384,ecdh-sha2-nistp521,diffie-hellman-group-exchange-sha256,diffie-hellman-group-exchange-sha1,diffie-hellman-group14-sha1,ext-info-c
4416 09:56:51 884 debug2: host key algorithms: ssh-rsa-cert-v01@openssh.com,rsa-sha2-512,rsa-sha2-256,ssh-rsa,ecdsa-sha2-nistp256-cert-v01@openssh.com,ecdsa-sha2-nistp384-cert-v01@openssh.com,ecdsa-sha2-nistp521-cert-v01@openssh.com,ssh-ed25519-cert-v01@openssh.com,ecdsa-sha2-nistp256,ecdsa-sha2-nistp384,ecdsa-sha2-nistp521,ssh-ed25519
4416 09:56:51 884 debug2: ciphers ctos: chacha20-poly1305@openssh.com,aes128-ctr,aes192-ctr,aes256-ctr,aes128-gcm@openssh.com,aes256-gcm@openssh.com,aes128-cbc,aes192-cbc,aes256-cbc,3des-cbc
4416 09:56:51 884 debug2: ciphers stoc: chacha20-poly1305@openssh.com,aes128-ctr,aes192-ctr,aes256-ctr,aes128-gcm@openssh.com,aes256-gcm@openssh.com,aes128-cbc,aes192-cbc,aes256-cbc,3des-cbc
4416 09:56:51 884 debug2: MACs ctos: umac-64-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512-etm@openssh.com,hmac-sha1-etm@openssh.com,umac-64@openssh.com,umac-128@openssh.com,hmac-sha2-256,hmac-sha2-512,hmac-sha1
4416 09:56:51 884 debug2: MACs stoc: umac-64-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512-etm@openssh.com,hmac-sha1-etm@openssh.com,umac-64@openssh.com,umac-128@openssh.com,hmac-sha2-256,hmac-sha2-512,hmac-sha1
4416 09:56:51 884 debug2: compression ctos: none,zlib@openssh.com,zlib
4416 09:56:51 884 debug2: compression stoc: none,zlib@openssh.com,zlib
4416 09:56:51 884 debug2: languages ctos: 
4416 09:56:51 884 debug2: languages stoc: 
4416 09:56:51 884 debug2: first_kex_follows 0 
4416 09:56:51 884 debug2: reserved 0 
4416 09:56:51 884 debug1: kex: algorithm: curve25519-sha256@libssh.org
4416 09:56:51 884 debug1: kex: host key algorithm: ssh-rsa
4416 09:56:51 884 debug1: kex: client->server cipher: chacha20-poly1305@openssh.com MAC: <implicit> compression: none
4416 09:56:51 884 debug1: kex: server->client cipher: chacha20-poly1305@openssh.com MAC: <implicit> compression: none
4416 09:56:51 884 debug1: expecting SSH2_MSG_KEX_ECDH_INIT
4416 09:56:51 884 debug3: w32_select fd:3
4416 09:56:51 884 debug3: Total in fds:1
4416 09:56:51 884 debug2: on_select - io:000000000025AC40 type:1 rd:1
4416 09:56:51 884 debug3: WSARecv - pio: 000000000025AC40
4416 09:56:51 884 debug2: WSARecv - reported IO pending
4416 09:56:51 884 debug3: wait() on 0 events and 0 children
4416 09:56:51 900 debug3: wait() on 0 events and 0 children
4416 09:56:51 900 debug2: WSARecvCompletionCB - io:000000000025AC40, pending_state:1, flags:0, error:0, received:48
4416 09:56:51 900 debug3: select - returning 1
4416 09:56:51 900 debug3: w32_read fd:3
4416 09:56:51 900 debug3: recv - io:000000000025AC40
4416 09:56:51 900 debug2: recv - returning 48 bytes from prior completed IO, remaining:0, io:000000000025AC40
4416 09:56:51 900 debug2: set_newkeys: mode 1
4416 09:56:51 900 debug1: SSH2_MSG_NEWKEYS sent
4416 09:56:51 900 debug1: expecting SSH2_MSG_NEWKEYS
4416 09:56:51 900 debug3: w32_write fd:3
4416 09:56:51 900 debug2: send - io:000000000025AC40
4416 09:56:51 900 debug2: send - WSASend() returned 0, APC scheduled io:000000000025AC40
4416 09:56:51 900 debug2: WSASendCB - io:000000000025AC40, pending_state:1, error:0, sent:624 of remaining:624
4416 09:56:51 900 debug3: w32_select fd:3
4416 09:56:51 900 debug3: Total in fds:1
4416 09:56:51 900 debug2: on_select - io:000000000025AC40 type:1 rd:1
4416 09:56:51 900 debug3: WSARecv - pio: 000000000025AC40
4416 09:56:51 900 debug2: WSARecv - reported IO pending
4416 09:56:51 900 debug3: wait() on 0 events and 0 children
4416 09:56:51 900 debug3: wait() on 0 events and 0 children
4416 09:56:51 900 debug2: WSARecvCompletionCB - io:000000000025AC40, pending_state:1, flags:0, error:0, received:16
4416 09:56:51 900 debug3: select - returning 1
4416 09:56:51 900 debug3: w32_read fd:3
4416 09:56:51 900 debug3: recv - io:000000000025AC40
4416 09:56:51 900 debug2: recv - returning 16 bytes from prior completed IO, remaining:0, io:000000000025AC40
4416 09:56:51 900 debug2: set_newkeys: mode 0
4416 09:56:51 900 debug1: SSH2_MSG_NEWKEYS received
4416 09:56:51 900 debug1: KEX done
4416 09:56:51 900 debug3: w32_select fd:3
4416 09:56:51 900 debug3: Total in fds:1
4416 09:56:51 900 debug2: on_select - io:000000000025AC40 type:1 rd:1
4416 09:56:51 900 debug3: WSARecv - pio: 000000000025AC40
4416 09:56:51 900 debug2: WSARecv - reported IO pending
4416 09:56:51 900 debug3: wait() on 0 events and 0 children
4416 09:56:51 900 debug3: wait() on 0 events and 0 children
4416 09:56:52 118 debug2: WSARecvCompletionCB - io:000000000025AC40, pending_state:1, flags:0, error:0, received:44
4416 09:56:52 118 debug3: select - returning 1
4416 09:56:52 118 debug3: w32_read fd:3
4416 09:56:52 118 debug3: recv - io:000000000025AC40
4416 09:56:52 118 debug2: recv - returning 44 bytes from prior completed IO, remaining:0, io:000000000025AC40
4416 09:56:52 118 debug3: w32_write fd:3
4416 09:56:52 118 debug2: send - io:000000000025AC40
4416 09:56:52 118 debug2: send - WSASend() returned 0, APC scheduled io:000000000025AC40
4416 09:56:52 118 debug2: WSASendCB - io:000000000025AC40, pending_state:1, error:0, sent:44 of remaining:44
4416 09:56:52 118 debug3: w32_select fd:3
4416 09:56:52 118 debug3: Total in fds:1
4416 09:56:52 118 debug2: on_select - io:000000000025AC40 type:1 rd:1
4416 09:56:52 118 debug3: WSARecv - pio: 000000000025AC40
4416 09:56:52 118 debug2: WSARecv - WSARecv() returned 0, io:000000000025AC40
4416 09:56:52 118 debug3: wait() on 0 events and 0 children
4416 09:56:52 118 debug2: WSARecvCompletionCB - io:000000000025AC40, pending_state:1, flags:0, error:0, received:68
4416 09:56:52 118 debug3: select - returning 1
4416 09:56:52 118 debug3: w32_read fd:3
4416 09:56:52 118 debug3: recv - io:000000000025AC40
4416 09:56:52 118 debug2: recv - returning 68 bytes from prior completed IO, remaining:0, io:000000000025AC40
4416 09:56:52 118 debug1: userauth-request for user myuserid service ssh-connection method none
4416 09:56:52 118 debug1: attempt 0 failures 0
4416 09:56:52 118 debug3: w32_getsockname fd:3
4416 09:56:52 118 debug3: w32_getsockname fd:3
4416 09:56:52 118 debug2: parse_server_config: config reprocess config len 331
4416 09:56:52 181 debug2: input_userauth_request: setting up authctxt for myuserid
4416 09:56:52 181 debug2: input_userauth_request: try method none
4416 09:56:52 181 Failed none for myuserid from 10.23.16.105 port 54396 ssh2
4416 09:56:52 181 debug3: userauth_finish: failure partial=0 next methods="publickey,password,keyboard-interactive"
4416 09:56:52 181 debug3: w32_write fd:3
4416 09:56:52 181 debug2: send - io:000000000025AC40
4416 09:56:52 181 debug2: send - WSASend() returned 0, APC scheduled io:000000000025AC40
4416 09:56:52 181 debug2: WSASendCB - io:000000000025AC40, pending_state:1, error:0, sent:76 of remaining:76
4416 09:56:52 181 debug3: w32_select fd:3
4416 09:56:52 181 debug3: Total in fds:1
4416 09:56:52 181 debug2: on_select - io:000000000025AC40 type:1 rd:1
4416 09:56:52 181 debug3: WSARecv - pio: 000000000025AC40
4416 09:56:52 181 debug2: WSARecv - reported IO pending
4416 09:56:52 181 debug3: wait() on 0 events and 0 children
4416 09:56:52 181 debug3: wait() on 0 events and 0 children
4416 09:56:52 181 debug2: WSARecvCompletionCB - io:000000000025AC40, pending_state:1, flags:0, error:0, received:364
4416 09:56:52 181 debug3: select - returning 1
4416 09:56:52 181 debug3: w32_read fd:3
4416 09:56:52 181 debug3: recv - io:000000000025AC40
4416 09:56:52 181 debug2: recv - returning 364 bytes from prior completed IO, remaining:0, io:000000000025AC40
4416 09:56:52 181 debug1: userauth-request for user myuserid service ssh-connection method publickey
4416 09:56:52 181 debug1: attempt 1 failures 0
4416 09:56:52 181 debug2: input_userauth_request: try method publickey
4416 09:56:52 181 debug1: test whether pkalg/pkblob are acceptable
4416 09:56:52 181 debug3: w32_write fd:3
4416 09:56:52 181 debug2: send - io:000000000025AC40
4416 09:56:52 181 debug2: send - WSASend() returned 0, APC scheduled io:000000000025AC40
4416 09:56:52 181 debug2: WSASendCB - io:000000000025AC40, pending_state:1, error:0, sent:324 of remaining:324
4416 09:56:52 181 debug2: userauth_pubkey: authenticated 0 pkalg ssh-rsa
4416 09:56:52 181 Postponed publickey for myuserid from 10.23.16.105 port 54396 ssh2
4416 09:56:52 181 debug3: w32_select fd:3
4416 09:56:52 181 debug3: Total in fds:1
4416 09:56:52 181 debug2: on_select - io:000000000025AC40 type:1 rd:1
4416 09:56:52 181 debug3: WSARecv - pio: 000000000025AC40
4416 09:56:52 181 debug2: WSARecv - reported IO pending
4416 09:56:52 181 debug3: wait() on 0 events and 0 children
4416 09:56:52 181 debug3: wait() on 0 events and 0 children
4416 09:56:52 181 debug2: WSARecvCompletionCB - io:000000000025AC40, pending_state:1, flags:0, error:0, received:644
4416 09:56:52 181 debug3: select - returning 1
4416 09:56:52 181 debug3: w32_read fd:3
4416 09:56:52 181 debug3: recv - io:000000000025AC40
4416 09:56:52 181 debug2: recv - returning 644 bytes from prior completed IO, remaining:0, io:000000000025AC40
4416 09:56:52 181 debug1: userauth-request for user myuserid service ssh-connection method publickey
4416 09:56:52 181 debug1: attempt 2 failures 0
4416 09:56:52 181 debug2: input_userauth_request: try method publickey
4416 09:56:52 181 debug3: w32_write fd:4
4416 09:56:52 181 debug2: write - io:00000000002595D0
4416 09:56:52 181 debug3: wait() on 0 events and 0 children
4416 09:56:52 181 debug2: WriteCB - pio:00000000002595D0, pending_state:1, error:0, transferred:4 of remaining: 4
4416 09:56:52 181 debug2: write - reporting 4 bytes written, io:00000000002595D0
4416 09:56:52 181 debug3: w32_write fd:4
4416 09:56:52 181 debug2: write - io:00000000002595D0
4416 09:56:52 181 debug3: wait() on 0 events and 0 children
4416 09:56:52 181 debug2: WriteCB - pio:00000000002595D0, pending_state:1, error:0, transferred:956 of remaining: 956
4416 09:56:52 181 debug2: write - reporting 956 bytes written, io:00000000002595D0
4416 09:56:52 181 debug3: w32_read fd:4
4416 09:56:52 181 debug3: read - io:00000000002595D0 remaining:0
4416 09:56:52 181 debug2: ReadFileEx io:00000000002595D0
4416 09:56:52 181 debug3: wait() on 0 events and 0 children
4416 09:56:53 947 debug2: ReadCB pio:00000000002595D0, pending_state:1, error:0, received:5
4416 09:56:53 947 debug2: read - io:00000000002595D0 read: 4 remaining: 1
4416 09:56:53 947 debug3: w32_read fd:4
4416 09:56:53 947 debug3: read - io:00000000002595D0 remaining:1
4416 09:56:53 947 debug2: read - io:00000000002595D0 read: 1 remaining: 0
4416 09:56:53 947 debug1: auth agent did not authorize client myuserid
4416 09:56:53 947 debug2: userauth_pubkey: authenticated 0 pkalg ssh-rsa
4416 09:56:53 947 Failed publickey for myuserid from 10.23.16.105 port 54396 ssh2: RSA SHA256:g/NZaQYcc3Ue5hf+WbTjaQ+g+2HCYFYNWSTPn6qd4QI
4416 09:56:53 947 debug3: userauth_finish: failure partial=0 next methods="publickey,password,keyboard-interactive"
4416 09:56:53 947 debug3: w32_write fd:3
4416 09:56:53 947 debug2: send - io:000000000025AC40
4416 09:56:53 947 debug2: send - WSASend() returned 0, APC scheduled io:000000000025AC40
4416 09:56:53 947 debug2: WSASendCB - io:000000000025AC40, pending_state:1, error:0, sent:76 of remaining:76
4416 09:56:53 947 debug3: w32_select fd:3
4416 09:56:53 947 debug3: Total in fds:1
4416 09:56:53 947 debug2: on_select - io:000000000025AC40 type:1 rd:1
4416 09:56:53 947 debug3: WSARecv - pio: 000000000025AC40
4416 09:56:53 947 debug2: WSARecv - reported IO pending
4416 09:56:53 947 debug3: wait() on 0 events and 0 children
4416 09:56:53 947 debug3: wait() on 0 events and 0 children
4416 09:56:53 947 debug2: WSARecvCompletionCB - io:000000000025AC40, pending_state:1, flags:0, error:0, received:92
4416 09:56:53 947 debug3: select - returning 1
4416 09:56:53 947 debug3: w32_read fd:3
4416 09:56:53 947 debug3: recv - io:000000000025AC40
4416 09:56:53 947 debug2: recv - returning 92 bytes from prior completed IO, remaining:0, io:000000000025AC40
4416 09:56:53 947 debug1: userauth-request for user myuserid service ssh-connection method keyboard-interactive
4416 09:56:53 947 debug1: attempt 3 failures 1
4416 09:56:53 947 debug2: input_userauth_request: try method keyboard-interactive
4416 09:56:53 947 debug1: keyboard-interactive devs 
4416 09:56:53 947 debug1: auth2_challenge: user=myuserid devs=
4416 09:56:53 947 debug1: kbdint_alloc: devices ''
4416 09:56:53 947 debug2: auth2_challenge_start: devices 
4416 09:56:53 947 Failed keyboard-interactive for myuserid from 10.23.16.105 port 54396 ssh2
4416 09:56:53 947 debug3: userauth_finish: failure partial=0 next methods="publickey,password,keyboard-interactive"
4416 09:56:53 947 debug3: w32_write fd:3
4416 09:56:53 947 debug2: send - io:000000000025AC40
4416 09:56:53 947 debug2: send - WSASend() returned 0, APC scheduled io:000000000025AC40
4416 09:56:53 947 debug2: WSASendCB - io:000000000025AC40, pending_state:1, error:0, sent:76 of remaining:76
4416 09:56:53 947 debug3: w32_select fd:3
4416 09:56:53 947 debug3: Total in fds:1
4416 09:56:53 947 debug2: on_select - io:000000000025AC40 type:1 rd:1
4416 09:56:53 947 debug3: WSARecv - pio: 000000000025AC40
4416 09:56:53 947 debug2: WSARecv - reported IO pending
4416 09:56:53 947 debug3: wait() on 0 events and 0 children
4416 09:56:53 947 debug3: wait() on 0 events and 0 children
6308 09:57:08 681 debug1: Request received (4)
6308 09:57:08 681 debug1: Request received (4)

```

**Comment by fpqc** [comment]
This sounds like you're just failing to authenticate. This works in Win10.

**Comment by DavidBiesack** [comment]
yes, it is failing to authenticate with my public key, and falls back to password. But I don't know why it fails, and the log does not explain it. I have verified that on the remote Windows PC, my `~/.ssh/authorized_keys` contains my public key from the local client's `id_rsa.pub` and this works on other Linux machines.  Below is a ssh -vvv log (I masked the hostname/userid),
but the important lines appear to be near the end:
```
debug3: sign_and_send_pubkey: RSA SHA256:g/NZaQYcc3Ue5hf+WbTjaQ+g+2HCYFYNWSTPn6qd4QI
debug3: send packet: type 50
debug3: receive packet: type 51
debug1: Authentications that can continue: publickey,password,keyboard-interactive
debug1: Authentications that can continue: publickey,password,keyboard-interactive
debug2: we did not send a packet, disable method   
```

but I don't know how to diagnose this. What does `we did not send a packet, disable method ` mean and why did the handshake fail?

Full log below. 
```

OpenSSH_7.2p2 Ubuntu-4ubuntu2.1, OpenSSL 1.0.2g  1 Mar 2016
debug1: Reading configuration data /u/userid/.ssh/config
debug1: /u/userid/.ssh/config line 9: Applying options for *
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: /etc/ssh/ssh_config line 19: Applying options for *
debug2: resolving "aaaaaaaaa.bbbbbbb.sas.com" port 22
debug2: ssh_connect_direct: needpriv 0
debug1: Connecting to aaaaaaaaa.bbbbbbb.sas.com [10.23.11.92] port 22.
debug1: Connection established.
debug1: identity file /u/userid/.ssh/id_rsa type 1
debug1: key_load_public: No such file or directory
debug1: identity file /u/userid/.ssh/id_rsa-cert type -1
debug1: Enabling compatibility mode for protocol 2.0
debug1: Local version string SSH-2.0-OpenSSH_7.2p2 Ubuntu-4ubuntu2.1
debug1: Remote protocol version 2.0, remote software version OpenSSH_7.1p1 Microsoft_Win32_port_with_VS
debug1: match: OpenSSH_7.1p1 Microsoft_Win32_port_with_VS pat OpenSSH* compat 0x04000000
debug2: fd 3 setting O_NONBLOCK
debug1: Authenticating to aaaaaaaaa.bbbbbbb.sas.com:22 as 'userid'
debug3: hostkeys_foreach: reading file "/u/userid/.ssh/known_hosts"
debug3: record_hostkey: found key type ECDSA in file /u/userid/.ssh/known_hosts:421
debug3: load_hostkeys: loaded 1 keys from aaaaaaaaa.bbbbbbb.sas.com
debug3: order_hostkeyalgs: prefer hostkeyalgs: ecdsa-sha2-nistp256-cert-v01@openssh.com,ecdsa-sha2-nistp384-cert-v01@openssh.com,ecdsa-sha2-nistp521-cert-v01@openssh.com,ecdsa-sha2-nistp256,ecdsa-sha2-nistp384,ecdsa-sha2-nistp521
debug3: send packet: type 20
debug1: SSH2_MSG_KEXINIT sent
debug3: receive packet: type 20
debug1: SSH2_MSG_KEXINIT received
debug2: local client KEXINIT proposal
debug2: KEX algorithms: curve25519-sha256@libssh.org,ecdh-sha2-nistp256,ecdh-sha2-nistp384,ecdh-sha2-nistp521,diffie-hellman-group-exchange-sha256,diffie-hellman-group-exchange-sha1,diffie-hellman-group14-sha1,ext-info-c
debug2: host key algorithms: ecdsa-sha2-nistp256-cert-v01@openssh.com,ecdsa-sha2-nistp384-cert-v01@openssh.com,ecdsa-sha2-nistp521-cert-v01@openssh.com,ecdsa-sha2-nistp256,ecdsa-sha2-nistp384,ecdsa-sha2-nistp521,ssh-ed25519-cert-v01@openssh.com,ssh-rsa-cert-v01@openssh.com,ssh-ed25519,rsa-sha2-512,rsa-sha2-256,ssh-rsa
debug2: ciphers ctos: chacha20-poly1305@openssh.com,aes128-ctr,aes192-ctr,aes256-ctr,aes128-gcm@openssh.com,aes256-gcm@openssh.com,aes128-cbc,aes192-cbc,aes256-cbc,3des-cbc
debug2: ciphers stoc: chacha20-poly1305@openssh.com,aes128-ctr,aes192-ctr,aes256-ctr,aes128-gcm@openssh.com,aes256-gcm@openssh.com,aes128-cbc,aes192-cbc,aes256-cbc,3des-cbc
debug2: MACs ctos: umac-64-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512-etm@openssh.com,hmac-sha1-etm@openssh.com,umac-64@openssh.com,umac-128@openssh.com,hmac-sha2-256,hmac-sha2-512,hmac-sha1
debug2: MACs stoc: umac-64-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512-etm@openssh.com,hmac-sha1-etm@openssh.com,umac-64@openssh.com,umac-128@openssh.com,hmac-sha2-256,hmac-sha2-512,hmac-sha1
debug2: compression ctos: none,zlib@openssh.com,zlib
debug2: compression stoc: none,zlib@openssh.com,zlib
debug2: languages ctos: 
debug2: languages stoc: 
debug2: first_kex_follows 0 
debug2: reserved 0 
debug2: peer server KEXINIT proposal
debug2: KEX algorithms: curve25519-sha256@libssh.org,ecdh-sha2-nistp256,ecdh-sha2-nistp384,diffie-hellman-group-exchange-sha256,diffie-hellman-group14-sha1
debug2: host key algorithms: ssh-rsa,ecdsa-sha2-nistp256,ssh-ed25519
debug2: ciphers ctos: chacha20-poly1305@openssh.com,aes128-ctr,aes192-ctr,aes256-ctr
debug2: ciphers stoc: chacha20-poly1305@openssh.com,aes128-ctr,aes192-ctr,aes256-ctr
debug2: MACs ctos: umac-64-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512-etm@openssh.com,hmac-sha1-etm@openssh.com,umac-64@openssh.com,umac-128@openssh.com,hmac-sha2-256,hmac-sha2-512,hmac-sha1
debug2: MACs stoc: umac-64-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512-etm@openssh.com,hmac-sha1-etm@openssh.com,umac-64@openssh.com,umac-128@openssh.com,hmac-sha2-256,hmac-sha2-512,hmac-sha1
debug2: compression ctos: none,zlib@openssh.com
debug2: compression stoc: none,zlib@openssh.com
debug2: languages ctos: 
debug2: languages stoc: 
debug2: first_kex_follows 0 
debug2: reserved 0 
debug1: kex: algorithm: curve25519-sha256@libssh.org
debug1: kex: host key algorithm: ecdsa-sha2-nistp256
debug1: kex: server->client cipher: chacha20-poly1305@openssh.com MAC: <implicit> compression: none
debug1: kex: client->server cipher: chacha20-poly1305@openssh.com MAC: <implicit> compression: none
debug3: send packet: type 30
debug1: expecting SSH2_MSG_KEX_ECDH_REPLY
debug3: receive packet: type 31
debug1: Server host key: ecdsa-sha2-nistp256 SHA256:QUUPbxvXH/5JGCTSzgj1LTzzwHyN/n8AcqC0lnjV1GA
debug3: hostkeys_foreach: reading file "/u/userid/.ssh/known_hosts"
debug3: record_hostkey: found key type ECDSA in file /u/userid/.ssh/known_hosts:421
debug3: load_hostkeys: loaded 1 keys from aaaaaaaaa.bbbbbbb.sas.com
debug3: hostkeys_foreach: reading file "/u/userid/.ssh/known_hosts"
debug3: record_hostkey: found key type ECDSA in file /u/userid/.ssh/known_hosts:422
debug3: load_hostkeys: loaded 1 keys from 10.23.11.92
debug1: Host 'aaaaaaaaa.bbbbbbb.sas.com' is known and matches the ECDSA host key.
debug1: Found key in /u/userid/.ssh/known_hosts:421
debug3: send packet: type 21
debug2: set_newkeys: mode 1
debug1: rekey after 134217728 blocks
debug1: SSH2_MSG_NEWKEYS sent
debug1: expecting SSH2_MSG_NEWKEYS
debug3: receive packet: type 21
debug2: set_newkeys: mode 0
debug1: rekey after 134217728 blocks
debug1: SSH2_MSG_NEWKEYS received
debug2: key: /u/userid/.ssh/id_rsa (0x5636f4870a10), explicit, agent
debug3: send packet: type 5
debug3: receive packet: type 6
debug2: service_accept: ssh-userauth
debug1: SSH2_MSG_SERVICE_ACCEPT received
debug3: send packet: type 50
debug3: receive packet: type 51
debug1: Authentications that can continue: publickey,password,keyboard-interactive
debug3: start over, passed a different list publickey,password,keyboard-interactive
debug3: preferred publickey,keyboard-interactive,password
debug3: authmethod_lookup publickey
debug3: remaining preferred: keyboard-interactive,password
debug3: authmethod_is_enabled publickey
debug1: Next authentication method: publickey
debug1: Offering RSA public key: /u/userid/.ssh/id_rsa
debug3: send_pubkey_test
debug3: send packet: type 50
debug2: we sent a publickey packet, wait for reply
debug3: receive packet: type 60
debug1: Server accepts key: pkalg ssh-rsa blen 279
debug2: input_userauth_pk_ok: fp SHA256:g/NZaQYcc3Ue5hf+WbTjaQ+g+2HCYFYNWSTPn6qd4QI
debug3: sign_and_send_pubkey: RSA SHA256:g/NZaQYcc3Ue5hf+WbTjaQ+g+2HCYFYNWSTPn6qd4QI
debug3: send packet: type 50
debug3: receive packet: type 51
debug1: Authentications that can continue: publickey,password,keyboard-interactive
debug2: we did not send a packet, disable method
debug3: authmethod_lookup keyboard-interactive
debug3: remaining preferred: password
debug3: authmethod_is_enabled keyboard-interactive
debug1: Next authentication method: keyboard-interactive
debug2: userauth_kbdint
debug3: send packet: type 50
debug2: we sent a keyboard-interactive packet, wait for reply
debug3: receive packet: type 51
debug1: Authentications that can continue: publickey,password,keyboard-interactive
debug3: userauth_kbdint: disable: no info_req_seen
debug2: we did not send a packet, disable method
debug3: authmethod_lookup password
debug3: remaining preferred: 
debug3: authmethod_is_enabled password
debug1: Next authentication method: password
userid@aaaaaaaaa.bbbbbbb.sas.com's password: 
debug3: send packet: type 50
debug2: we sent a password packet, wait for reply
debug3: receive packet: type 52
debug1: Authentication succeeded (password).
Authenticated to aaaaaaaaa.bbbbbbb.sas.com ([10.23.11.92]:22).
```

**Comment by manojampalam** [comment]
5924 09:56:53 947 debug1: unable to generate token for user myuserid

This is the reason it pubkey auth is failing. What is the user account type - domain or workgroup?

If domain, try out 
    ssh user@domain@IP
If workgroup, make sure ssh-lsa.dll is installed
    Check step 8 in https://github.com/PowerShell/Win32-OpenSSH/wiki/Install-Win32-OpenSSH

**Comment by DavidBiesack** [comment]
Yes, thank you, this works:
```
ssh userid@domainid@hostname remote-command
```

I did find https://github.com/PowerShell/Win32-OpenSSH/wiki/ssh.exe-examples but only after your reply above which prompted me to search for 'domain'. As a (primarily) Linux user where 'domain' is not part of the ssh environment, this is not very obvious, so perhaps the documentation can be clearer. For Win -> Win ssh, most users probably understand this.

thanks.

**Comment by manojampalam** [comment]
I assume the following works too (as detailed in https://github.com/PowerShell/Win32-OpenSSH/wiki/ssh.exe-examples)
ssh.exe user@domain host

**Comment by DavidBiesack** [comment]
Close. Needs the `-l` option as listed on that page.

```
ssh.exe -l user@domain host
```

**Comment by v4virus4u** [comment]
I am using OpenSSH_7.4p1, OpenSSL 1.0.2d 9 Jul 2015 & able to do a password-less ssh connection from linux to the windows system.

Within windows localhost am not able to perform password-less ssh connection using:
ssh.exe localhost 'command'
<command executes & output is displayed on console>

But if I specify a user say admin/root, a password prompt appears:
ssh.exe Administrator@localhost 'command'
'domain'\Administrator@localhost's password:

My project requires a password-less connection irrespective of user. Appreciate the guidance!

Thanks!
